### ISSUE TITLE
Russian translation update

### ISSUE THREAD

**Issue by einhander** [akiko-pusu/redmine_issue_templates#340]
Hello, Russian translation needs to be updated here the new file.
[ru.yml.txt](https://github.com/akiko-pusu/redmine_issue_templates/files/4554072/ru.yml.txt)

**Comment by akiko-pusu** [comment]
```
# Russian strings go here for Rails i18n
ru:
  issue_templates: Шаблоны
  issue_template: Шаблон задачи
  issue_template_note: Заметка
  label_enabled: Вкл.
  label_disabled: Выкл.
  label_help_message: Подсказка по шаблонам
  label_show_help_message: Показывать подсказку по шаблонам при создаии/редактировании задачи.
  about_help_message: У каждого проекта может быть своя подсказа по шаблонам
  close_help: Закрыть подсказку
  about_template_help_message: Вы можете посмотреть подсказку по работе с шаблонами для данного проекта
  label_enabled_help_message: Если параметр установлен, шаблон доступен для использования в задаче. Отключите, если хотите сохранить шаблон как черновик.
  label_list_templates: "Список шаблонов"
  label_new_templates: "Добавить шаблон"
  issue_template_name: "Название шаблона"
  issue_description: "Шаблон текста задачи"
  issue_title: "Заголовок задачи"
  label_applied_for_issue: "Заполняемые поля задачи: "
  help_for_issue_title: "Если установлено значение для 'Заголовок задачи', то оно будет автоматически вставляться в Тему задачи при выборе шаблона."
  help_for_this_field: "Справка по данному полю"
  permission_manage_issue_templates: "Управление шаблонами"
  permission_edit_issue_templates: "Редактировать шаблоны"
  permission_show_issue_templates: "Показывать шаблоны"
  project_module_issue_templates: "Использовать шаблоны"
  label_isdefault_help_message: "Если параметр установлен, шаблон будет подставляться автоматически при создании новой задачи для соответствующего трекера."
  defaulf_template_loaded: "В поле описания задачи подставлен шаблон по умолчанию (Трекер: %{tracker})"
  text_no_tracker_enabled: "Для проекта не настроен ни один трекер.\nПожалуйста, настройте трекеры, так как шаблоны должны быть ассоциированы с трекерами."
  label_enabled_sharing: "Разрешить использование шаблона в дочерних проектах."
  label_inherit_templates: "Унаследовать шаблоны родительского проекта"
  label_inherit_templates_help_message: "Наследует шаблоны, настроенные для родительского проекта. Унаследованы будут только те шаблоны, для которых это разрешено в параметрах шаблона (настройка Разрешить использование шаблона во всем дереве проектов)."
  label_inherited_templates: "Унаследованные шаблоны"
  no_issue_templates_for_this_project: "Для данного проекта не настроены шаблоны задач."
  link_to_index_edit_template: "Список шаблонов задач / редактирование шаблонов"
  erase_issue_subject_and_description: "Очистить тему и описание задачи."
  unused_tracker_at_this_project: "ВНИМАНИЕ: Данный трекер не используется в текущем проекте. Пожалуйста перенастройте шаблон при необходимости."
  label_enabledshaing_help_message: "Если параметр установлен, данный шаблон может быть унаследован дочерними проектами. (Для работы данной функции необходимо включить наследование в настройках шаблонов дочернего проекта.)"
  label_should_replaced: "Заменять тему и описание"
  label_should_replaced_help_message: "Если параметр установлен, текст темы и описания будет очищен и заменен текстом из шаблона. （По умолчанию параметр отключен, и текст добавляется к существующему, а не заменяется.)"
  global_issue_templates: "Глобальные шаблоны задач"
  no_issue_templates_for_this_redmine: "Глобальные шаблоны задач, доступные для всех проектов Redmine, не настроены."
  only_admin_can_associate_global_template: "Подключить глобальные шаблоны задач к данному проекту может только администратор Redmine."
  text_no_tracker_enabled_for_global: "Не настроен ни один трекер.\nПожалуйста, настройте трекеры, так как шаблоны должны быть ассоциированы с трекерами."
  display_and_filter_issue_templates_in_dialog: "Подобрать шаблон"
  label_filter_template: "Подобрать шаблон"
  label_apply_global_template_to_all_projects: "Применить глобальный шаблон ко всем проектам"
  label_use_template_when_edit: "Использовать шаблон при редактировании"
  note_template: "Шаблон комментариев"
  orphaned_template: "Потерянные шаблоны"
  project_list_associated_this_template: "Проекты с этим шаблоном"
  help_project_local_template_override_global_template: "Если эта опция активирована, то глобальные шаблоны будут назначены на все проекты, если проект имеет свой шаблон, то глобальный шаблон будет спрятан при создании новой задачи."
  warning_project_local_template_override_global_template: "Пожалуйста будте осторожны, т.к. глобальные шаблоны могут содержать конфиденциальную информацию и они назначаются на все проекты."
  warning_enable_builtin_fields: "Внимание эта функция экспериментальная, используйте на свой страх и риск."
  help_enable_builtin_fields: "При включении будет показана форма, позволяющая выставить шаблоны для встроенных или пользовтельских полей. Пожалуйста зарегистрируейте настройки в формате JSON."
  no_note_templates_for_this_project: "Для проекта еще не назначены шаблоны комментариев." 
```

**Comment by akiko-pusu** [comment]
Hi, @einhander 
Thanks for your translation file!
I've just merged into my master branch (1.0.x) and 0.3-stable branch.

Stay safe and best wishes!
